src/whereInTheJson.js

```diff
@@ -0,0 +1,51 @@
+var weirdObject = `{
+    "name": "William Shakespeare",
+    "dead" : true,
+    "works" : [
+        {
+            "name" : "Romeo and Juliet",
+            "published" : 1591,
+            "isAwesome" : true
+        },
+        {
+            "name" : "Richard III",
+            "published" : 1592,
+            "isAwesome" : false
+        }
+    ],
+    "favoriteSites" : [
+        "tumblr",
+        "4chan"
+    ]
+}`
+var jsonObject = JSON.parse(weirdObject)
+
+export default function whereInTheJson(obj, inputValue){
+
+
+  for (let key in obj) {
+    var val = obj[key]
+    if (val === inputValue) {
+      return key
+    }
+
+    if (Array.isArray(val) === true) {
+      for (var i = 0; i < val.length; i++) {
+        if (val[i] === inputValue) {
+          var path =  key + " -> " + val.indexOf(val[i])
+          return path
+        }
+      }
+//not working:
```

Comment:
Thanks for letting me know, I did try to run the test and it keeps returning undefined.